File: 1791-richest-customer-wealth/richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int n = accounts.length;
        int m = accounts[0].length;
        int max = 0;
        for(int i=0; i<n; i++){
            int result = 0;
            for(int j=0; j<m; j++){
                result += accounts[i][j];
            }
            max = Math.max(result, max);
        }
        return max;
    }
}

File: 1791-richest-customer-wealth/richest-customer-wealth.py
class Solution(object):
    def maximumWealth(self, accounts):
        """
        :type accounts: List[List[int]]
        :rtype: int
        """
        max_wealth = 0
        for customer in accounts:
            wealth = sum(customer)
            max_wealth = max(wealth, max_wealth)
        return max_wealth

        